src/main/java/org/humancellatlas/ingest/security/SecurityConfig.java

```diff
@@ -13,38 +16,50 @@
 import org.springframework.security.web.util.matcher.AntPathRequestMatcher;
 
 import javax.servlet.http.HttpServletRequest;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
-import java.util.Optional;
+import java.util.*;
 import java.util.stream.Stream;
 
 import static java.util.stream.Collectors.toList;
 import static org.humancellatlas.ingest.security.ElixirConfig.ELIXIR;
 import static org.humancellatlas.ingest.security.GcpConfig.GCP;
-import static org.humancellatlas.ingest.security.Role.GUEST;
-import static org.humancellatlas.ingest.security.Role.WRANGLER;
+import static org.humancellatlas.ingest.security.Role.*;
 import static org.springframework.http.HttpMethod.*;
 
 @EnableWebSecurity
 @Configuration
 public class SecurityConfig extends WebSecurityConfigurerAdapter {
-
-    private static final String FORWARDED_FOR = "X-Forwarded-For";
+    private static final String FORWARDED_HOST = "x-forwarded-host";
```

Comment:
@rdgoite the x-forwarded-for is missing i'm not sure why so I used x-forwarded-host. I know this was working before i'm not sure why it's null now. Any thoughts?